### PR TITLE
Chair pose

### PR DESCRIPTION
**Pull request by jesslns**
Add isChairPose function to posenetWrapper

### CODE REVIEW

File: dist/megaScript.js [hunk 1]
```diff
@@ -0,0 +1 @@
+!function(e){var t={};function r(n){if(t[n])return t[n].exports;var o=t[n]={i:n,l:!1,exports:{}};return e[n].call(o.exports,o,o.exports,r),o.l=!0,o.exports}r.m=e,r.c=t,r.d=function(e,t,n){r.o(e,t)||Object.defineProperty(e,t,{enumerable:!0,get:n})},r.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},r.t=function(e,t){if(1&t&&(e=r(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var n=Object.create(null);if(r.r(n),Object.defineProperty(n,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var o in e)r.d(n,o,function(t){return e[t]}.bind(null,o));return n},r.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return r.d(t,"a",t),t},r.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},r.p="",r(r.s=0)}([function(e,t){}]);
```
Comment (punchcafe) [inline dist/megaScript.js]: Why is this here?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,15757 @@
+{
```
Comment (punchcafe) [inline package-lock.json]: what is this and why is it part of this PR?

File: spec/posenet_sample/chair_pose_testing.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
+// good resources: https://www.yogabaron.com/yoga-poses
+
```
Comment (punchcafe) [inline spec/posenet_sample/chair_pose_testing.js]: looks good

File: src/components/checklist/__snapshots__/checklist.test.js.snap [hunk 4]
```diff
@@ -11,7 +11,12 @@ exports[`Checklist should render correctly with no props 1`] = `
         "isStatic": true,
```
Comment (punchcafe) [inline src/components/checklist/__snapshots__/checklist.test.js.snap]: don't know if this will still pass after merge

File: src/lib/chair.test.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import sample from '../../spec/posenet_sample/chair_pose_testing';
+import Pose from './posenetWrapper';
```
Comment (punchcafe) [inline src/lib/chair.test.js]: good!

File: src/lib/posenetWrapper.js [hunk 6]
```diff
@@ -59,52 +59,113 @@ class Pose {
...
 
   isMountainPose() {
-    // var conditional =  new conditional([
-    //this._isPointBetween(blah),
-    //this._isHigherThan(blah),
-    //this._isLowerThan(blah),
-    //])
-    // return conditional.isMet
-    const criteria_1 = "Right hand below hip";
+      // var conditional =  new conditional([
```
Comment (punchcafe) [inline src/lib/posenetWrapper.js]: this isn't the concern of this PR

File: src/lib/posenetWrapper.js [hunk 7]
```diff
@@ -59,52 +59,113 @@ class Pose {
...
+          [check_1, criteria_1],
+          [check_2, criteria_2],
+          [check_3, criteria_3],
+          [check_4, criteria_4],
+          [check_5, criteria_5]
+        ]
+      ];
+    }
+
+  isChairPose(){
```
Comment (punchcafe) [inline src/lib/posenetWrapper.js]: good!

**Comment by jesslns** [comment]
Fixed issues by making a new branch.
